src/components/themed/ModalParts.tsx

```diff
@@ -39,7 +42,11 @@ export function ModalMessage(props: { children: React.ReactNode; paddingRem?: nu
   const styles = getStyles(theme)
   const padding = sidesToPadding(mapSides(fixSides(paddingRem, 0), theme.rem))
 
-  return <Text style={[styles.messageText, padding, isWarning && styles.warningText]}>{children}</Text>
+  return (
+    <EdgeText style={[styles.messageText, padding, isWarning && styles.warningText]} disableFontScaling numberOfLines={0}>
+      {children}
+    </EdgeText>
```

Comment:
Nope on this one. Instead, copy over the androidAdjust logic.

The `EdgeText` component has a *HUGE* problem - it forces `numberOfLines` to be a number. This is fine for labels and titles and such, where we don't want text arbitrarily wrapping. It is super-bad for arbitrary messages, where we want the whole thing to be reada...

Long-term, we need two text components. `EdgeText` should be renamed `LabelText`, and `ModalMessage` should be renamed to `MessageText`. Then devs will grab either one based whether they are doing a paragraph or a line.

In the short-term, we can't risk introducing the cut-off text bug to all our modals.

Reply:
Thanks for the info, good to know